<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBox15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox0">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider0">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
